game-core/src/main/java/games/strategy/triplea/ui/BattlePanel.java

```diff
@@ -184,7 +184,7 @@ private boolean ensureBattleIsDisplayed(final GUID battleId) {
 
         new IllegalStateException(
             "battle not displayed, looking for:" + battleId + " showing:" + currentBattleDisplayed)
-            .printStackTrace();
+                .printStackTrace();
```

Comment:
Wow this is super ugly

Reply:
Yup, saw that, too.  There's a few other `printStackTrace()` calls left in the codebase.  I have a TODO to replace them with `Logger`.